code/script.js

```diff
@@ -2,18 +2,171 @@
-)
 
+alert(`Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`);
+const userName = prompt("What is your name?");
+
+alert(`Wellcome to Technigo pizzaria, ${userName}`);
+
+///////////////////////////////////////////////////////////////////////////////
 // Step 2 - Food choice
 // Your code goes here
 
+const foodTypeNum = parseInt(
+  prompt(
+    `What type of food would you like to eat?
+   Enter a number:
+    1 - Pizza 
+    2 - Salad 
+    3 - Pasta`
+  )
+);
+
+let foodType;
+
+switch (foodTypeNum) {
+  case 1:
+    foodType = "Pizza";
+    break;
+  case 2:
+    foodType = "Salad";
+    break;
+  case 3:
+    foodType = "Pasta";
+    break;
+  default:
+    alert("Invalid choice. Please select a number between 1 and 3");
+    process.exsit(1);
+}
+
+// confirming what type of food you want to eat
+
+alert(`Would you want to eat ${foodType}? Click "ok" to confirm`);
+
+///////////////////////////////////////////////////////////////////////////////
 // Step 3 - Subtype choice
 // Your code goes here
 
+let dishNum;
+let yourDish;
+
+if (foodType === "Pizza") {
+  dishNum = parseInt(
+    prompt(
+      `Which food would like to order? \n Enter a number:
+       1 - Vegan Pizza
+       2 - Vegan Cheese Pizza 
+       3 - Vegetarian Pizza`
+    )
+  );
+
+  switch (dishNum) {
+    case 1:
+      yourDish = " Vegan Pizza";
+      break;
+    case 2:
+      yourDish = "Vegan Cheese Pizza";
+      break;
+    case 3:
+      yourDish = "Vevetarian Pizza";
+      break;
+    default:
+      alert("Invalid choice. Please select a number between 1 and 3");
+      process.exsit(1);
+  }
+} else if (foodType === "Salad") {
+  dishNum = parseInt(
+    prompt(
+      `Which food would like to order? 
+       Enter a number: 
+       1 - Vegan Feta Salad
+       2 - Italian Salad 
+       3 - Grilled Vegan Cheese Salad`
+    )
+  );
+
+  switch (dishNum) {
+    case 1:
+      yourDish = " Vegan Feta Salad";
+      break;
+    case 2:
+      yourDish = "Italian Salad";
+      break;
+    case 3:
+      yourDish = "Grilled Vegan Cheese Salad";
+      break;
+    default:
+      alert("Invalid choice. Please select a number between 1 and 3");
+      process.exsit(1);
+  }
+} else if (foodType === "Pasta") {
+  dishNum = parseInt(
+    prompt(
+      `Which food would like to order? 
+      Enter a number: 
+      1 - Tomato Pasta
+      2 - Vegan Cream Pasta 
+      3 - Ruccola and Dryed Tomato Pasta`
+    )
+  );
+
+  switch (dishNum) {
+    case 1:
+      yourDish = " Tomato Pasta";
+      break;
+    case 2:
+      yourDish = "Vegan cream Pasta";
+      break;
+    case 3:
+      yourDish = "Ruccola and Dryed Tomato Pasta";
+      break;
+    default:
+      alert("Invalid choice. Please select a number between 1 and 3");
+      process.exsit(1);
+  }
+}
+
+// confirming which dish you want to eat
+
+alert(`You've chosen ${yourDish}!  - Click "ok" to confirm`);
```

Comment:
This is a really minor thing, but I noticed there are occasionally backticks and occasionally double quotations for example inside alert parenthesis. I'm wondering if there is a reason to sometimes use one and sometimes the other?